spec/requests/packages_spec.rb

```diff
@@ -460,33 +460,6 @@
     end
   end
 
-  describe 'getting a package with included vendor' do
```

Comment:
These were the only places I had any uncertainty about. I'm not sure if these tests needed to be removed or if they should have been changed to test for `?include=provider` instead.